pkg/util/fs_utils.go

```diff
@@ -53,6 +54,24 @@ func GetSize(path string) int64 {
 	return stat.Size()
 }
 
+//GetFileContents returns the contents of a file at the specified path
+func GetFileContents(path string) ([]string, error) {
+	stat, err := os.Stat(path)
```

Comment:
Is this checking for the existence of a file?

It should be 
```
if _, err := os.Stat("/path/to/whatever"); os.IsNotExist(err) {
  // path/to/whatever does not exist
}
```

Reply:
it does check for existence, but also checks that the path isn't a directory after, which is why I thought it might be easier to split it up?

Reply:
I guess what is the expected behavior when the file doesn't exist?  I think you wouldn't want to throw an error?
Different scenarios:


**A**: "" 
**B**: no file
**output**: ?

**A**: "something" 
**B**: no file
**output**: ?

**A**: no file
**B**: no file
**output**: ?